changelog/std-traits-issomestring.dd

```diff
@@ -0,0 +1,88 @@
+$(REF `isSomeString`, std, traits) and $(REF `isNarrowString`, std, traits) are now `false` for enums
```

Comment:
Don't use `$(REF ...)` in the summary line. The whole line is turned into a link. REF messes with that.